config/testdata/user-impersonation/patch.yaml

```diff
@@ -0,0 +1,12 @@
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          args:
+          - --events-addr=http://notification-controller/
```

Comment:
Please remove all these args and use only those specified here https://github.com/fluxcd/kustomize-controller/blob/main/config/manager/deployment.yaml#L44-L47